lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -304,8 +304,13 @@ def _section_student_admin(course, access):
         'get_student_progress_url_url': reverse('get_student_progress_url', kwargs={'course_id': unicode(course_key)}),
         'enrollment_url': reverse('students_update_enrollment', kwargs={'course_id': unicode(course_key)}),
         'reset_student_attempts_url': reverse('reset_student_attempts', kwargs={'course_id': unicode(course_key)}),
+        'reset_student_attempts_for_entrance_exam_url': reverse('reset_student_attempts_for_entrance_exam',
```

Comment:
Follow the line breaking advice from here:

https://github.com/edx/edx-platform/wiki/Python-Guidelines